src/App.tsx

```diff
@@ -12,155 +14,41 @@ export const App = () => (
     >
       <div className="container">
```

Comment:
I would create Navigation component, not only NavLink, to make your code clearer and readable.